### PR TITLE
add data base

### PR DESCRIPTION
**Pull request by miguelriverasarmiento**
hello, mount the database and save the emails
Captura de pantalla de 2021-02-28 17-39-24

### CODE REVIEW

File: package.json [hunk 1]
```diff
@@ -22,9 +22,10 @@
     "connect-multiparty": "^2.2.0",
     "ejs": "^3.1.5",
     "express": "^4.17.1",
-    "mongodb": "^3.6.4",
-    "mongoose": "^5.11.15",
-    "morgan": "^1.10.0"
+    "method-override": "^3.0.0",
```
Comment (jjmedinas) [inline package.json]: What is this for?

File: package.json [hunk 2]
```diff
@@ -22,9 +22,10 @@
...
+    "mongoose": "^5.11.17",
```
Comment (jjmedinas) [inline package.json]: If you're going to use postgres now, you should uninstall mongo and its dependencies.

File: src/controller/emailsController.js [hunk 3]
```diff
@@ -0,0 +1,55 @@
...
+            console.log(error)
+        }
+        res.status(201).send(`User added with ID: ${results}`)
+    })
+}
+
+module.exports = {
+    createUser,
+}
+/*var mongoose = require('mongoose');
```
Comment (jjmedinas) [inline src/controller/emailsController.js]: Please delete commented code if you're are not using it.

File: src/routes/index.js [hunk 4]
```diff
@@ -1,9 +1,11 @@
...
-router.get('/', (req, res) => {
+router.get('/', function(req, res) {
     res.render('index.html', { title: 'First web'});
   });
 
-  module.exports = router;
\ No newline at end of file
+router.post('/', emailControllerPost.createUser);
+
+module.exports = router;
```
Comment (jjmedinas) [inline src/routes/index.js]: Please add a blank at the end of the file, it's good practice.

File: src/model/user.js [hunk 5]
```diff
@@ -0,0 +1,10 @@
+'use strict'
```
Comment (jjmedinas) [inline src/model/user.js]: How are you using this?

File: src/index.js [hunk 6]
```diff
@@ -1,22 +1,21 @@
...
-
+const methodOverride = require("method-override");
+//var User = require('./model/user');
 // Settings
 app.set('port', 4000);
 app.set('views', path.join(__dirname, 'views'));
 app.engine('html', require('ejs').renderFile);
 app.set('view engine', 'ejs');
 
+app.use(bodyParser.urlencoded({ extended: false }));
```
Comment (jjmedinas) [inline src/index.js:12]: What are you doing in this line?

**Review by jjmedinas — CHANGES_REQUESTED** [review]

File: package.json (hunk 1 above)
Comment (miguelriverasarmiento) [reply package.json]: That's garbage, I forgot to erase and simplify the code, I will correct it and practice git

File: src/model/user.js (hunk 5 above)
Comment (miguelriverasarmiento) [reply src/model/user.js]: That is in the javascript course, but I'm also going to remove it

File: src/index.js (hunk 6 above)
Comment (miguelriverasarmiento) [reply src/index.js:12]: I just read that it is not necessary for the express version, I will delete it

File: src/routes/index.js (hunk 4 above)
Comment (miguelriverasarmiento) [reply src/routes/index.js]: ok I do it

File: src/controller/emailsController.js (hunk 3 above)
Comment (miguelriverasarmiento) [reply src/controller/emailsController.js]: yes, I forgot it too, I will delete it

**Review by jjmedinas — APPROVED** [review]
Good job!! 👏 🚀

**Pull request merged**